### PR TITLE
Clean up smoketest testApps

### PR DESCRIPTION
**Pull request by littleaj**
There were a few apps that were either unused or not smoketest apps in the smoketest directory. We need to be more strict about what we include under /test/smoke/ for the changes I want to make in the future.

### CODE REVIEW

File: test/webapps/PerfTestApp/build.gradle [hunk 1]
```diff
@@ -1,7 +1,9 @@
 apply plugin: 'war'
 
+def aiSdkVersion = version
+
 dependencies {
-    compile aiCoreJar
+    compile "com.microsoft.azure:applicationinsights-core:$aiSdkVersion"
```
Comment (trask) [inline test/webapps/PerfTestApp/build.gradle]: Can we use `$version` here directly? I think it's understood when working in multi-module project that all versions are the same.

File: test/webapps/PerfTestAppSpringBoot/build.gradle [hunk 2]
```diff
@@ -17,8 +17,10 @@ compileJava.targetCompatibility = 1.7
 compileSmokeTestJava.sourceCompatibility = 1.8
 compileSmokeTestJava.targetCompatibility = 1.8
 
+def aiSdkVersion = version
+
 dependencies {
-    compile springBootStarterJar
+    compile "com.microsoft.azure:applicationinsights-spring-boot-starter:$aiSdkVersion"
```
Comment (trask) [inline test/webapps/PerfTestAppSpringBoot/build.gradle]: Same question above about using `$version` directly here

File: test/webapps/PerfTestApp/build.gradle (hunk 1 above)
Comment (littleaj) [reply test/webapps/PerfTestApp/build.gradle]: I wanted to have the option to use a different version than the one in the repo.

**Comment by littleaj** [comment]
@trask, I think my autocrlf=true (checkout `\r\n`, check-in `\n`). I can change it to as-is if you prefer.

**Comment by trask** [comment]
It looks like we had some files with bad line endings stored in git from long ago and the line endings got fixed when you moved them 👍

**Comment by trask** [comment]
@littleaj it looks this has merge conflicts due to #975

**Pull request merged**